app/src/test/java/com/antosik/benchproject/app/common/binding/swipeRefreshLayout/SwipeRefreshLayoutBindingAdaptersKtTest.kt

```diff
@@ -0,0 +1,18 @@
+package com.antosik.benchproject.app.common.binding.swipeRefreshLayout
+
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+import io.mockk.mockk
+import org.amshove.kluent.shouldBe
+import org.junit.jupiter.api.Test
+
+internal class SwipeRefreshLayoutBindingAdaptersKtTest {
+
+    val swipeRefreshLayout: SwipeRefreshLayout = mockk(relaxed = true)
```

Comment:
You are testing mockk not your implementation 😄 you should never do this because there is no benefit of testing the mocks.

Reply:
Done :D